gravatar/build.gradle.kts

```diff
@@ -84,6 +84,7 @@ dependencies {
     testImplementation(libs.mockk.android)
     testImplementation(libs.mockk.agent)
     testImplementation(libs.kotlinx.coroutines.test)
+    testImplementation(kotlin("test"))
```

Comment:
Testing the inside of the expected exception with [JUnit 4 is not very straightforward](https://www.baeldung.com/junit-assert-exception#1-assert-exception-is-thrown), so adding this dependency here [makes our life easier.](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-fails-with.html)